docs/usage/trad-cluster.md

```diff
@@ -0,0 +1,18 @@
+# Traditional Clusters
+
+Traditional clusters, or simply "clusters" in this document. Are machines that you can provision and run yourself.
```

Comment:
Should this be just one sentence?  "Traditional clusters, or simply "clusters" in this document, are machines that you can provision and run yourself."